### PR TITLE
initial integration tests

### PR DESCRIPTION
**Pull request by marciodaitan**
Implemented initial functional tests for the home and contact pages.

### CODE REVIEW

File: test/app.test.js [hunk 1]
```diff
@@ -1,18 +1,137 @@
+/* jshint esversion: 6 */
```
Comment (gvillalta99) [inline test/app.test.js]: @marciodaitan can we configure this option on `.jshintrc`?

**Comment by gvillalta99** [comment]
@marciodaitan can you review the README.md file if the instructions to run the tests are correct?

**Comment by marciodaitan** [comment]
The instructions are correct, we can run tests either via the npm command or by running mocha directly.

**Comment by gvillalta99** [comment]
Thank you @marciodaitan!
Let's merge this!

**Pull request merged**